### ISSUE TITLE
flaggie will be removed from the portage tree on 2023-01-24.

### ISSUE THREAD

**Issue by babykart** [edannenberg/kubler#220]
```
- app-portage/flaggie-0.2.1-r2::gentoo (masked by: package.mask)
/var/db/repos/gentoo/profiles/package.mask:
# Michał Górny <mgorny@gentoo.org> (2022-12-25)
# make.conf writing is broken and package.use support incomplete.
# Last release in 2013.  Attempted unsuccessfully fixing it in 2017.
# Use an editor instead.
# Removal on 2023-01-24.  Bug #888423.
```

**Comment by edannenberg** [comment]
Thanks for the heads up! I would have take a closer look at the listed bugs, but if it doesn't affect Kubler usage we could also just move the Flaggie ebuild to our overlay.

**Comment by babykart** [comment]
The new flaggie is in the portage tree.

**Comment by r7l** [comment]
@babykart Am i seeing it wrong? The version in Portage seems to be from 2016.

```
* app-portage/flaggie
     Available versions:  0.2.1-r2 {PYTHON_TARGETS="python3_8 python3_9 python3_10"}
     Homepage:            https://github.com/projg2/flaggie/
     Description:         A smart CLI mangler for package.* files
```

Which looks like this version to me: https://github.com/projg2/flaggie/releases/tag/flaggie-0.2.1

**Comment by babykart** [comment]
@r7l you are seeing it right.
Strange, but the actual ebuild is not masked anymore.
